deploy/operator.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  name: portworx-operator
+  namespace: kube-system
+  initializers:
+    pending: []
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
```

Comment:
This annotation is being deprecated in 1.14. `priorityClassName` should be used instead. Also, I'm not sure if we need this for the operator itself. This is usually required to prevent eviction for pods that can affect the operation of a cluster if it is evicted. The operator will come up on another node in this case without any negative effect.